lib/uktest/tests.ld

```diff
@@ -9,4 +9,4 @@ SECTIONS
 		uk_asserttab_end = .;
 	}
 }
-INSERT AFTER .rodata;
+INSERT AFTER .data;
```

Comment:
I tested it with `.uk_inittab` (test run in x86_64 and aarch64), but probably any of the sections between `_etext` and `_rodata` is fine.
```suggestion
INSERT AFTER .uk_inittab;
```

Reply:
Sorry, I don't understand how this fixes initial problem. From my point of view, we could go with two solutions 1) roll back to PAGE_SIZE alignment solution 2) change multiboot_header : load_end_addr should point to _bss_start roughly

Reply:
After a short discussion: I agree with @alexhoppus that after `.data` is a good place for this section and the multiboot header should get the start of the `.bss` section as `load_end_addr` (option 2). The `.uk_inittab` tables are also read-only, it is only that they are currently not protected as they have no memory region in `plat/kvm/memory.c`.

Maybe outside the scope of this PR: I think in the future it would be a good idea to mark specific locations in the linkerscript where extra read-writable and read-only data section can safely placed by linkerscripts includes.

Reply:
Sorry, for not jumping into this discussion earlier. I did test the change and it worked for me. But yeah, different configurations may lead to different results. This is really not optimal. I agree @kubanrob.

Actually, for changing the multiboot header I already have a fix internally as I had a similar issue some time ago (though not with the uktest section). I can provide the patch as PR, if you like.